sqlalchemy_redshift/__init__.py

```diff
@@ -11,6 +11,25 @@
 __version__ = get_distribution('sqlalchemy-redshift').version
 
 from sqlalchemy.dialects import registry
+from sqlalchemy.dialects.postgresql.base import PGDialect
+
+def _get_server_version_info(self, connection):
+    v = connection.exec_driver_sql("select pg_catalog.version()").scalar()
+    m = re.match(
```

Comment:
In integration tests, this function does get invoked, but throws:

```
       NameError: name 're' is not defined
```

Reply:
Thank you! I added now.